src/wasm-exts/values.ml

```diff
@@ -0,0 +1,86 @@
+(*
+This module originated as a copy of interpreter/syntax/values.ml in the
+reference implementation.
+With adjustments from memory 64.
```

Comment:
I wonder what was adjusted...

Reply:
Yes, there is no specific memory64-adjustment. I only included `values.ml` to keep the import of `open Values` in the other modules of this library. The header text is just a "copy-paste" result from "types.ml" etc. I will adjust it.